example/src/ListWithHeader/Header.tsx

```diff
@@ -11,6 +11,7 @@ import Animated, {
   useSharedValue,
   withTiming,
 } from 'react-native-reanimated';
+import { COLORS } from 'src/common';
```

Comment:
Why not

```suggestion
import { COLORS } from '../common';
```

?

Reply:
VSCode does weird things when auto-importing